autoflake.py

```diff
@@ -569,9 +607,11 @@ def _main(argv, standard_out, standard_error):
                         help='by default, only unused standard library '
                              'imports are removed; specify a comma-separated '
                              'list of additional modules/packages')
+    parser.add_argument('--expand-single-star-import', action='store_true',
```

Comment:
Also, I think it would better match the other options if it were plural (`--expand-star-imports`)